scripts/skill_auditor/test_metrics_extractor.py

```
@@ -0,0 +1,513 @@
+        tmp_path = Path(tmp)
+        skill_md = tmp_path / "SKILL.md"
+        skill_md.write_text("---\nname: test\n---\n")
+
+        # Make file unreadable
+        skill_md.chmod(0o000)
+
+        try:
+            try:
+                extract_skill_metrics(tmp_path)
+                # If we get here, the test fails - no exception was raised
+                assert False, "Expected PermissionError to be raised but it was not"
```

Ruff (0.14.5)

Assertion always fails, replace with `pytest.fail()`

(PT015)